### PR TITLE
cowpatty: deparallelize

### LINKED ISSUE #45150: OSX yosemite: 'cowpatty' formula installation error
Hi!

OSX 10.10.5. Installing cowpatty:

```
$ brew install cowpatty -v
==> Downloading http://www.willhackforsushi.com/code/cowpatty/4.6/cowpatty-4.6.tgz
Already downloaded: /Library/Caches/Homebrew/cowpatty-4.6.tgz
==> Verifying cowpatty-4.6.tgz checksum
tar xf /Library/Caches/Homebrew/cowpatty-4.6.tgz
==> make BINDIR=/usr/local/Cellar/cowpatty/4.6/bin install
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb   -c -o md5.o md5.c
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb   -c -o sha1.o sha1.c
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb   -c -o utils.o utils.c
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb   -c -o cowpatty.o cowpatty.c
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb   -c -o genpmk.o genpmk.c
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb cowpatty.c -o cowpatty utils.o md5.o sha1.o -lpcap -lcrypto
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb genpmk.c -o genpmk utils.o sha1.o -lpcap -lcrypto
clang: error: no such file or directory: 'utils.o'
clang: error: no such file or directory: 'sha1.o'
make: *** [genpmk] Error 1
make: *** Waiting for unfinished jobs....
clang: error: no such file or directory: 'utils.o'
clang: error: no such file or directory: 'md5.o'
clang: error: no such file or directory: 'sha1.o'
make: *** [cowpatty] Error 1
==> Formula
Path: /usr/local/Library/Formula/cowpatty.rb
==> Configuration
HOMEBREW_VERSION: 0.9.5
ORIGIN: https://github.com/Homebrew/homebrew.git
HEAD: 3c9da0212be124c6aa7d2e14ed1c13b14767f01e
Last commit: 4 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: 12-core 64-bit ivybridge
OS X: 10.10.5-x86_64
Xcode: 7.0.1
CLT: 7.0.0.0.1.1441394355
Clang: 7.0 build 700
X11: 2.7.7 => /opt/X11
System Ruby: 2.0.0-p481
Perl: /usr/bin/perl
Python: /usr/local/bin/python => /usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /usr/local/var/rbenv/shims/ruby => /usr/local/var/rbenv/versions/2.1.6/bin/ruby
Java: 1.8.0_60
==> ENV
HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
MAKEFLAGS: -j12
CMAKE_PREFIX_PATH: /usr/local/opt/openssl:/usr/local
CMAKE_INCLUDE_PATH: /usr/include/libxml2:/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
PKG_CONFIG_PATH: /usr/local/opt/openssl/lib/pkgconfig
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/usr/local/Library/ENV/pkgconfig/10.10
ACLOCAL_PATH: /usr/local/share/aclocal
PATH: /usr/local/Library/ENV/4.3:/usr/local/opt/openssl/bin:/usr/bin:/bin:/usr/sbin:/sbin

Error: cowpatty 4.6 did not build
Logs:
     /Users/drewshg312/Library/Logs/Homebrew/cowpatty/01.make
     /Users/drewshg312/Library/Logs/Homebrew/cowpatty/01.make.cc
```

**/Users/drewshg312/Library/Logs/Homebrew/cowpatty/01.make:**
```
2015-10-19 10:58:13 -0700

make
BINDIR=/usr/local/Cellar/cowpatty/4.6/bin
install

clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb   -c -o md5.o md5.c
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb   -c -o sha1.o sha1.c
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb   -c -o utils.o utils.c
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb   -c -o cowpatty.o cowpatty.c
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb   -c -o genpmk.o genpmk.c
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb cowpatty.c -o cowpatty utils.o md5.o sha1.o -lpcap -lcrypto
clang -pipe -Wall -DOPENSSL  -O2 -g3 -ggdb genpmk.c -o genpmk utils.o sha1.o -lpcap -lcrypto
clang: error: no such file or directory: 'utils.o'
clang: error: no such file or directory: 'md5.o'
clang: error: no such file or directory: 'sha1.o'
clang: error: no such file or directory: 'utils.o'
clang: error: no such file or directory: 'sha1.o'
make: *** [cowpatty] Error 1
make: *** Waiting for unfinished jobs....
make: *** [genpmk] Error 1

HOMEBREW_VERSION: 0.9.5
ORIGIN: https://github.com/Homebrew/homebrew.git
HEAD: 3c9da0212be124c6aa7d2e14ed1c13b14767f01e
Last commit: 4 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: 12-core 64-bit ivybridge
OS X: 10.10.5-x86_64
Xcode: 7.0.1
CLT: 7.0.0.0.1.1441394355
GCC-4.2: build 0
Clang: 7.0 build 700
X11: 2.7.7 => /opt/X11
System Ruby: 2.0.0-p481
Perl: /usr/bin/perl
Python: /usr/bin/python
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 1.8.0_60

HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
MAKEFLAGS: -j12
CMAKE_PREFIX_PATH: /usr/local/opt/openssl:/usr/local
CMAKE_INCLUDE_PATH: /usr/include/libxml2:/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
PKG_CONFIG_PATH: /usr/local/opt/openssl/lib/pkgconfig
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/usr/local/Library/ENV/pkgconfig/10.10
ACLOCAL_PATH: /usr/local/share/aclocal
PATH: /usr/local/Library/ENV/4.3:/usr/local/opt/openssl/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

**/Users/drewshg312/Library/Logs/Homebrew/cowpatty/01.make.cc:**
```
clang called with: -pipe -Wall -DOPENSSL -O2 -g3 -ggdb -c -o genpmk.o genpmk.c
superenv removed:  -Wall -O2 -g3 -ggdb
superenv added:    -w -Os -march=native -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include
superenv executed: clang -pipe -w -Os -march=native -pipe -DOPENSSL -c -o genpmk.o genpmk.c -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include

clang called with: -pipe -Wall -DOPENSSL -O2 -g3 -ggdb -c -o md5.o md5.c
superenv removed:  -Wall -O2 -g3 -ggdb
superenv added:    -w -Os -march=native -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include
superenv executed: clang -pipe -w -Os -march=native -pipe -DOPENSSL -c -o md5.o md5.c -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include

clang called with: -pipe -Wall -DOPENSSL -O2 -g3 -ggdb -c -o sha1.o sha1.c
superenv removed:  -Wall -O2 -g3 -ggdb
superenv added:    -w -Os -march=native -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include
superenv executed: clang -pipe -w -Os -march=native -pipe -DOPENSSL -c -o sha1.o sha1.c -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include

clang called with: -pipe -Wall -DOPENSSL -O2 -g3 -ggdb -c -o cowpatty.o cowpatty.c
superenv removed:  -Wall -O2 -g3 -ggdb
superenv added:    -w -Os -march=native -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include
superenv executed: clang -pipe -w -Os -march=native -pipe -DOPENSSL -c -o cowpatty.o cowpatty.c -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include

clang called with: -pipe -Wall -DOPENSSL -O2 -g3 -ggdb cowpatty.c -o cowpatty utils.o md5.o sha1.o -lpcap -lcrypto
superenv removed:  -Wall -O2 -g3 -ggdb
superenv added:    -w -Os -march=native -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib -L/usr/local/lib -L/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries -Wl,-headerpad_max_install_names
superenv executed: clang -pipe -w -Os -march=native -pipe -DOPENSSL cowpatty.c -o cowpatty utils.o md5.o sha1.o -lpcap -lcrypto -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib -L/usr/local/lib -L/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries -Wl,-headerpad_max_install_names

clang called with: -pipe -Wall -DOPENSSL -O2 -g3 -ggdb -c -o utils.o utils.c
superenv removed:  -Wall -O2 -g3 -ggdb
superenv added:    -w -Os -march=native -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include
superenv executed: clang -pipe -w -Os -march=native -pipe -DOPENSSL -c -o utils.o utils.c -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include

clang called with: -pipe -Wall -DOPENSSL -O2 -g3 -ggdb genpmk.c -o genpmk utils.o sha1.o -lpcap -lcrypto
superenv removed:  -Wall -O2 -g3 -ggdb
superenv added:    -w -Os -march=native -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib -L/usr/local/lib -L/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries -Wl,-headerpad_max_install_names
superenv executed: clang -pipe -w -Os -march=native -pipe -DOPENSSL genpmk.c -o genpmk utils.o sha1.o -lpcap -lcrypto -isystem/usr/local/include -isystem/usr/include/libxml2 -isystem/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers -I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib -L/usr/local/lib -L/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries -Wl,-headerpad_max_install_names
```

### PR DESCRIPTION
**Pull request by apjanke**
Fixes #45150

### CODE REVIEW

File: Library/Formula/cowpatty.rb [hunk 1]
```diff
@@ -12,6 +12,8 @@ class Cowpatty < Formula
   end
 
   def install
+    # May fail on parallel builds with a race condition, see #45150
```
Comment (DomT4) [inline Library/Formula/cowpatty.rb]: Will upstream take a bug report on this at all?

Comment (apjanke) [inline Library/Formula/cowpatty.rb]: Maybe. I emailed the author a bug report using contact info on the website. Last update to cowpatty was 6 years ago, though, so who knows? Will let you know if they answer. Would be an easy `Makefile` fix, if they're still taking patches.

Comment (DomT4) [inline Library/Formula/cowpatty.rb]: It'd be good to get something upstream, but feel free to merge with the current comment if there's no public way of accessing the email. Probably worth pulling bottles as well, given the `make` stage.

**Comment by DomT4** [comment]
Will merge this to close the other issue and stop us seeing any further ones. If you need any help with `brew pull` or anything feel free to say, we're all happy to help :). Thanks Andrew!

**Pull request closed without merging**